src/ActiveSetMethods.jl

```diff
@@ -6,6 +6,8 @@ import MathOptInterface
 
 const MOI = MathOptInterface
 const MOIU = MathOptInterface.Utilities
+statistics = Dict()
```

Comment:
I think this would better be in `Model` struct.

Reply:
I am a bit confused by the scoping issue caused by this. Do you know any example for this?

Reply:
What is the scoping issue? I think it could be done similar to this: https://github.com/exanauts/ActiveSetMethods/blob/master/src/model.jl#L30

Reply:
But like `Parameters` struct, statistics whether as a struct or dictionary has to be defined somewhere outside `Model`